packages/ms-spectrum/src/getMatchingScore.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const { xSum } = require('ml-spectra-processing');
+
+/**
+ * Filter the array of peaks
+ * @param {object} [matchedExpFragments={}] - object of all the experimental fragments matched
+ * @param {Array} [fragmentsContribution=[]] - array of all the bond contributions for of matching fragments
+ * @param {object} [options={}] - object containing the scale factor for mass and intensity
+ * @param {number} [options.massCoefficient=3]
+ * @param {number} [options.intensityCoefficient=0.6]
+ * @returns {number} - returns the matching score
+ */
+
+// Scaling mass and intensity power can be found in DOI: 10.1016/1044-0305(94)87009-8
```

Comment:
You should not add comments here because there is a risk jsdoc will not be associated to the function